pkg/controller/route/route.go

```diff
@@ -824,7 +824,8 @@ func (rc *RouteController) sync(ctx context.Context, key string) error {
 				 */
 				var replicas int32 = 2
 				podLabels := map[string]string{
-					"app": tmpName,
+					"app":                  tmpName,
+					api.AcmeTemporaryLabel: "true",
```

Comment:
The temporary label is used for deletion of temporary objects - so it should match RS, but not pods owned by that RS. If that's needed for networkpolicies (I don't get to explore that yet), I'd be open toad another label to our objects.

Reply:
We can use another label!